scripts/ci-conformance.sh

```diff
@@ -45,8 +45,8 @@ export GCP_NETWORK_NAME=${GCP_NETWORK_NAME:-"${TEST_NAME}-mynetwork"}
 GCP_B64ENCODED_CREDENTIALS=$(base64 -w0 "$GOOGLE_APPLICATION_CREDENTIALS")
 export GCP_B64ENCODED_CREDENTIALS
 export KUBERNETES_MAJOR_VERSION="1"
-export KUBERNETES_MINOR_VERSION="23"
-export KUBERNETES_PATCH_VERSION="8"
+export KUBERNETES_MINOR_VERSION="24"
```

Comment:
No idea, but could/should this be 25? (similar in other places)

Reply:
ok